ui/admin/app/routes/scopes/scope/sessions.js

```diff
@@ -116,4 +117,14 @@ export default class ScopesScopeSessionsRoute extends Route {
   clearAllFilters() {
     this.status = [];
   }
+
+  /**
+   * refreshes session data.
+   */
+
+  @action
+  @loading
```

Comment:
I don't think we need the loading decorator here.  The call to refresh the route should trigger the loading state.